src/main/java/com/google/devtools/build/lib/bazel/BazelStartupOptionsModule.java

```diff
@@ -33,10 +33,17 @@
         valueHelp = "<path>",
         help =
             "The location of the user .bazelrc file containing default values of "
-                + "Bazel options. If unspecified, Bazel uses the first .bazelrc file it finds in "
+                + "Bazel options. This option can also be specified multiple times.\n"
+                + "E.g. with `--bazelrc=x.rc --bazelrc=y.rc`, options in both RCs will be read.\n"
+                + "--bazelrc values that come after `/dev/null` will be ignored to maintain "
```

Comment:
Rephrase to avoid the "backward compatibility" bit - in a few weeks/months/years it won't make sense anymore. Also, can you combine the info on the special value of `/dev/null` with the explanation below? Something like...

`/dev/null` indicates that all further `--bazelrc`s should be ignored. This is useful for disabling the search for a ...`

Reply:
Sounds good.

Reply:
revised.